glrd/manage.py

```diff
@@ -846,22 +903,31 @@ def diff_releases(existing_merged_releases, merged_releases):
                     for path, change in changes.items():
                         formatted_path = path.replace("root", "")
                         logging.info(f"{release_name} - {change_type}: {formatted_path} type changed from '{change['old_type']}' to '{change['new_type']}'")
-                else:
-                    for path, change in changes.items():
-                        formatted_path = path.replace("root", "")
-                        logging.info(f"{release_name} - {change_type}: {formatted_path}")
+                elif change_type in ['dictionary_item_added', 'dictionary_item_removed', 'iterable_item_added', 'iterable_item_removed']:
+                    changes_list = sorted(list(changes))
+                    for change in changes_list:
+                        formatted_change = change.replace("root", "")
 
 def save_output_file(data, filename, format="yaml"):
     """Save the data to a file in the specified format."""
     with open(filename, 'w') as file:
         if format == 'yaml':
-            yaml.dump(data, file, default_flow_style=False, sort_keys=False)
+            # Create a custom dumper class that doesn't generate anchors
```

Comment:
While the use is clear inline classes are always a bad thing as you can't reuse and can easily oversee them.

Reply:
Yepp, sure. Leftover from "trying it out". Moved to `util.py`.